components/database/mongodb-40/files/mongodb_cluster_40.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+
+This file and its contents are supplied under the terms of the
+Common Development and Distribution License ("CDDL)". You may
+only use this file in accordance with the terms of the CDDL.
+
+A full copy of the text of the CDDL should have accompanied this
+source. A copy of the CDDL is also available via the Internet at
+http://www.illumos.org/license/CDDL.
+
+Copyright 2012 EveryCity Ltd. All rights reserved.
+
+-->
+<service_bundle type="manifest" name="mongodb">
```

Comment:
I'd avoided delivering whole cluster configuration example. That's fine to deliver just mongos SMF configuration, but cluster configuration can be a) rather specific, b) it's easy to configure it on the base of existing configs for those who need it.